MaintenanceApi.yaml

```diff
@@ -384,6 +490,72 @@ components:
         - endTime
+    MaintenanceComponent:
+      type: object
+      properties:
+        uuid:
+          type: string
+          format: uuid
+          description: The identifier of this maintenance component. This identifier is inherited from/dictated by the FRAIHMWORK Health And Integrity API.
+                  
+        maintenanceState:
+          description: The maintenance state of the component.
+          $ref: '#/components/schemas/MaintenanceState'
+          
+        maintenanceStatistics:
+          $ref: '#/components/schemas/MaintenanceStatistics'
+      required: 
+        - uuid
+        - maintenanceState
+        - maintenanceStatistics        
+    MaintenanceState:
+      type: object
+      properties:
+        state:
+          type: string
+          description: "
+          This is a high level description of the component or subsystem state. <br> <br>
+          AVAILABLE describes a repairable system or system element that is operational at a given point in time <br> <br>
+          DOWN describes a repairable system or system element that is not operational at a given point in time, whether the cause of downtime has been discovered or the cause is in repair <br> <br>"
+          enum:
+          - AVAILABLE
+          - DOWN
+      required:
+        - state        
+    MaintenanceStatistics:
+      type: object
+      properties:
+        meanTimeBetweenFailureHours:
+          description: The mean of the times between failures, in hours. If a component has been running without downtime, the MTBF is equal to the duration between introduction to FRAIHMWORK and the current time. Otherwise the MTBF is equal to the mean of the completed dwells within AVAILABLE state.
+          type: number
+          format: double
+          
+        meanTimeToRecoveryHours:
+          description: The mean of the times to recover the failures, in hours. If a component has been running in its initial downtime, the MTTR is equal to the duration between introduction of the first repairable fault and the current time. Otherwise the MTTR is equal to the mean of the completed dwells within DOWN state. A recovery is the time at which all repairs are completed on a component.
+          type: number
+          format: double
+          
+        totalAvailableHours:
+          description: This is the amount of time that the component has been AVAILABLE since it has been registered with FRAIHMWORK. 
+          type: number
+          format: double
+          
+        totalDownHours:
+          description: This is the amount of time that the component has been DOWN since it has been registered with FRAIHMWORK. 
+          type: number
+          format: double
+
+        totalDownEntries:
+          description: This is the number of times the component has transitioned out of DOWN state due to clearing all repairable faults. 
+          type: integer
+          format: int64
+
+        totalAvailableEntries:
```

Comment:
Should be renamed to fit in line with the FRAIHM-1772 requirements, IE, something like "totalNumberOfRecoveries"